gtfs/spec/en/reference.md

```diff
@@ -198,7 +198,7 @@ Primary key (`route_id`)
 |  ------ | ------ | ------ | ------ |
 |  `route_id` | Unique ID | **Required** | Identifies a route. |
 |  `agency_id` | Foreign ID referencing `agency.agency_id` | **Conditionally Required** | Agency for the specified route.<br><br>Conditionally Required:<br>- **Required** if multiple agencies are defined in [agency.txt](#agency). <br>- Optional otherwise.   |
-|  `route_short_name` | Text | **Conditionally Required** | Short name of a route. Often a short, abstract identifier (e.g., "32", "100X", "Green") that riders use to identify a route. Both `route_short_name` and `route_long_name` may be defined.<br><br>Conditionally Required:<br>- **Required** if `routes.route_long_name` is empty.<br>- Optional otherwise. |
+|  `route_short_name` | Text | **Conditionally Required** | Short name of a route. Often a short, abstract identifier (e.g., "32", "100X", "Green") that riders use to identify a route. Both `route_short_name` and `route_long_name` may be defined.<br><br>Conditionally Required:<br>- **Required** if `routes.route_long_name` is empty.<br>- Recommended if there is a brief service designation. This should be the commonly-known passenger name of the service, no longer than 12 characters. |
```

Comment:
I understand the "recommended" is pretty much the same as "should" here. Not 100% sure but is it possible that people get confused when comparing the statement in `When is a file/field Recommended instead of Optional?` section of this PR? ("Recommended if...."vs "...in all circumstances)

Reply:
@tzujenchanmbd This is a good point! Recommended is used in pre-existing field descriptions outside this PR, e.g [calendar_dates file description](https://gtfs.org/schedule/reference/#calendar_datestxt), so maybe I should remove the statement in the PR since it isn't true. I am basically trying to get across "how does the **recommended presence value** differ from the **optional presence value**, but maybe that's an overly complex nuance. Any thoughts?

Reply:
Thanks for sharing this specificity in your feedback @tzujenchanmbd! I've redrafted the "When is a file/field Recommended instead of Optional?" section with #1 since I think it's better to be more explicit. Feel free to take a look and let me know if it's improved.

Reply:
@emmambd Yes it looks better to me, thanks!